src/tokens/hooks.ts

```diff
@@ -98,11 +99,20 @@ export function useCashInTokens() {
   return useSelector((state) => cashInTokensByNetworkIdSelector(state, networkIdsForCico))
 }
 
-export function useCashOutTokens() {
+export function useCashOutTokens(showZeroBalanceTokens: boolean = false) {
   const networkIdsForCico = getDynamicConfigParams(
     DynamicConfigs[StatsigDynamicConfigs.MULTI_CHAIN_FEATURES]
   ).showCico
-  return useSelector((state) => cashOutTokensByNetworkIdSelector(state, networkIdsForCico))
+  return useSelector((state) =>
+    cashOutTokensByNetworkIdSelector(state, networkIdsForCico, showZeroBalanceTokens)
+  )
+}
+
+export function useSpendTokens() {
```

Comment:
When enabling this feature flag in Statsig I see all tokens as enabled for spend; however, when disabled I only see `cUSD` and `cEUR` for Bidali.